doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -156,11 +156,13 @@ In order for the networking solution(s) to be conformant with the Reference Arch
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.ntw.001`|CNI multiplexer/metaplugin|As the selected CNI multiplexer/metapulgin MUST support other CNI plugins (`req.inf.ntw.06`) and SHOULD provide an API based solution to administer the networks from a central point (`req.inf.ntw.03`) the selected CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm).| [req.inf.ntw.06](./chapter02.md#23-kubernetes-architecture-requirements), [req.inf.ntw.03](./chapter02.md#23-kubernetes-architecture-requirements) |
-|`ra2.ntw.002`|CNI to implement a default network which implements the Kubernetes network model|[Calico](https://github.com/projectcalico/cni-plugin) may be used based on the requirement `req.inf.ntw.08` due to it's capability to handle `NetworkPolicies`|[req.inf.ntw.08](./chapter02.md#23-kubernetes-architecture-requirements)|
-|`ra2.ntw.003`|NAT less connectivity|IPVLAN CNI component of DANM or the [MACVLAN CNI](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan) may be used||
-|`ra2.ntw.004`|User plane networks|[User Space CNI](https://github.com/intel/userspace-cni-network-plugin) may be used. The User Space CNI may use VPP or OVS-DPDK as a backend.||
-|`ra2.ntw.005`|SR-IOV|SR-IOV CNI plugin and the [SR-IOV Device Plugin](https://github.com/intel/sriov-network-device-plugin) may be used||
+|`ra2.ntw.001`|CNI multiplexer/metaplugin|As the selected CNI multiplexer/metapulgin MUST support other CNI plugins (`req.inf.ntw.06`) and SHOULD provide an API based solution to administer the networks from a central point (`req.inf.ntw.03`) the selected CNI multiplexer/metapulgin must be [DANM](https://github.com/nokia/danm).| [req.inf.ntw.06](./chapter02.md#23-kubernetes-architecture-requirements), [req.inf.ntw.03](./chapter02.md#23-kubernetes-architecture-requirements) |
```

Comment:
How did DAMN become a "must" from a "may" -- before the restructuring exercise there was discussion and both MULTUS and DAMN were to be supported. Secondly, if we are changing the content in this PR then other changes, namely, the Plumbing group recommendations etc. can be out of scope of this PR.

Reply:
Back in Prague we agreed that we should harden the requirements as this the purpose of making an RA. I've forgot about the language change, that's why applied it in this change. If this does not fit into the restructuring work I can revert those changes. 

Yes there were discussions to support additional CNI multiplexers on top of DANM, but there was never a pr created to get an agreement on those changes.

Can you explain a bit more the last comment about the Plumbing group recommendations?

Reply:
We should keep restructuring free of changes with regards to requirement content or compliance statements such as "may", "must", .... It's otherwise getting tricky with reviews.

Reply:
Post Prague we have had discussion on this in a previously open PR including language suggested by Taylor.

The Table needs other rows -- as we need to add other requirements, for example, conformance to a standard CRD, support for any CNI plugin, etc. etc. 

From, [Network Plumbing Working Group](https://github.com/k8snetworkplumbingwg/community#network-plumbing-working-group)
`The Network Plumbing Working Group (NPWG) was formed at Kubecon NA '17 in Austin to initially work towards implementing a common standard addressing how one may attach multiple networks to pods in Kubernetes. Our work has culminated in a de-facto standard CRD.`

Reply:
Add DANM and Multus to the RI-2 vs the RA-2.  The RA-2 should be about the requirements which implementations such as DANM and Multus can provide solutions for.

Reply:
See above comments - I think we need to very explicitly document the pros/cons of each approach and document which we take. https://github.com/cntt-n/CNTT/pull/1854#discussion_r467806118